components/dropdowns/AdvancedDropDown.js

```diff
@@ -327,12 +327,12 @@ const AdvancedDropDown = (props) => {
                             multiSelect={true}
                             onSelect={addOption}
                         />
-                        <Button
-                            style={styles.submitButton}
-                            appearance={buttonAppearance}
-                            size='medium'
-                            icon={CheckIcon}
-                            onPress={() => submitField()}
+                        <Image
+//                            style={styles.submitButton}
+//                            appearance={buttonAppearance}
+//                            size='medium'
+//                            icon={CheckIcon}
+//                            onPress={() => submitField()}
```

Comment:
Go ahead and delete if everything functions correctly without the props. This applies to all of the images.